service/create/service.go

```diff
@@ -454,6 +449,64 @@ func (s *Service) Boot() {
 						return
 					}
 
+					// Create Ingress Load Balancer.
+					lbInput = LoadBalancerInput{
+						Name:        cluster.Spec.Cluster.Kubernetes.IngressController.Domain,
+						Clients:     clients,
+						Cluster:     cluster,
+						InstanceIDs: workerIDs,
+						PortsToOpen: []int{
+							httpPort,
```

Comment:
I think there is a ports problem. We need to open 80 and 443 but the listener ports need to be 30010 and 30011. These are the node ports for the ingress controller that are in clustertpr. 

See https://github.com/giantswarm/aws-operator/blob/master/service/create/ports.go#L16